File: leetcode2/1easy/이준열/3270.cpp
class Solution {
    public:
        int generateKey(int num1, int num2, int num3) {
            int num1Thous[4] = {0,};
            int num2Thous[4] = {0,};
            int num3Thous[4] = {0,};
            int p = 0;
            int output = 0;
            int div = 1000;
    
            while (div != 0)
            {
                num1Thous[p] = num1 / div;
                num2Thous[p] = num2 / div;
                num3Thous[p] = num3 / div;
                num1 -= num1Thous[p] * div;
                num2 -= num2Thous[p] * div;
                num3 -= num3Thous[p] * div;
                output += div * min(min(num1Thous[p],num2Thous[p]), num3Thous[p]);
    
                div /= 10;
                p++;
            }
            return output;
            
        }
    };

File: leetcode2/1easy/유시연/3270.py
<<<<<<< HEAD
class Leetcode.이재훈.이재훈.Solution:
=======
class Leetcode.이재훈.Solution:
>>>>>>> ab7f46f73d8f18bcd38c5cf06ecabae64f53a424
    def generateKey(self, num1: int, num2: int, num3: int) -> int:
        n1 = f"{num1:04d}"
        n2 = f"{num2:04d}"
        n3 = f"{num3:04d}"
    
        key_digits = []

        for i in range(4):
            min_digit = min(int(n1[i]), int(n2[i]), int(n3[i]))
            key_digits.append(str(min_digit))

        key_str = ''.join(key_digits)


        return int(key_str)